webapp/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "ecosystem-infra-rotation",
+  "version": "1.0.0",
+  "description": "Rotation tool for the Chrome Ecosystem Infra team",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/foolip/ecosystem-infra-rotation.git"
+  },
+  "license": "BSD-3-Clause",
```

Comment:
There actually is no license file in the repo, so maybe "UNLICENSED"?

Reply:
```suggestion
  "license": "UNLICENSED",
```